test/unit/helpers/JsonBuilder.ts

```diff
@@ -69,8 +69,9 @@ type BuildBoundaryEventParameter = {
   attachedToRef: string;
 } & BuildInterruptingEventParameter;
 
+export type BuildEventDefinition = 'message' | 'signal' | 'timer' | 'error' | 'escalation' | 'cancel' | 'compensate' | 'conditional' | 'link' | 'terminate'; // | 'multiple' | 'parallel-multiple' -- TODO: To implement later
```

Comment:
**nitpick:** generally we try to avoid adding new TODO.
Why not putting the values right away?

Reply:
Because I didn't think about implementation and wanted to implement what I needed for #2855

Reply:
```suggestion
export type BuildEventDefinition = 'message' | 'signal' | 'timer' | 'error' | 'escalation' | 'cancel' | 'compensate' | 'conditional' | 'link' | 'terminate'; 
```